dev/DynamicDependency/WinRTInprocModule.h

```diff
@@ -0,0 +1,120 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#if !defined(WINRTINPROCMODULE_H)
+#define WINRTINPROCMODULE_H
+
+#include <activationregistration.h>
+#include <activation.h>
+
+#include "MddWinRT.h"
+
+//TODO Components won't respect COM lifetime. workaround to get them in the COM list? See dev\UndockedRegFreeWinRT\catalog.cpp
+
+namespace MddCore
+{
+class WinRTInprocModule
+{
+public:
+    WinRTInprocModule() = default;
+
+    WinRTInprocModule(WinRTInprocModule&& other) :
+        m_path(std::move(other.m_path)),
+        m_dll(std::move(other.m_dll)),
+        m_dllGetActivationFactory(std::move(m_dllGetActivationFactory))
```

Comment:
this is a function pointer.  Is std::move correct here?

Reply:
std::move works for function pointers. Seems the moral equivalent of {} -- sure you can use other syntax but sometimes you need this, and always doing it is goodness for all the reasons {} initializers are goodness.

Logically we want to move the ptr from rhs to lhs, so lhs=std::move(rhs) is logically and technically correct.

BUT...it should be moving the value from other
```
m_dllGetActivationFactory(std::move(***other.***m_dllGetActivationFactory))
```
Doh! <sheepish grin>

Fixed